src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -204,7 +204,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Reflection\LoaderAllocator.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\MdConstant.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\MdFieldInfo.cs" />
-    <Compile Include="$(BclSourcesRoot)\System\Reflection\MdImport.cs" />
```

Comment:
Does Enum depend on MdImport anywhere, or is that unrelated change?

Reply:
@jkotas Enum uses CorElementType from that file, I guess other types in that file will be useful too soon.

Reply:
CorElementType in mono is a little bit different,e.g. `CorElementType.Boolean` instead of `CorElementType.ELEMENT_TYPE_BOOLEAN` in CoreCLR.

Reply:
@EgorBo extract CorElementType into its own file for now and keep the rest there